packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts

```diff
@@ -318,31 +324,74 @@ export class OdspDocumentDeltaConnection extends DocumentDeltaConnection {
     }
 
     public requestOps(from: number, to: number) {
-        this.getOpsCounter++;
-        const nonce = `${this.requestOpsNoncePrefix}${this.getOpsCounter}`;
+        this.pushCallCounter++;
+        const nonce = `${this.requestOpsNoncePrefix}${this.pushCallCounter}`;
+
+        // PUSH may disable this functionality
+        // back-compat: remove cast to any once latest version of IConnected is consumed
+        if ((this.details as any).supportedFeatures?.[feature_get_ops] !== true) {
```

Comment:
The flush method checks for supported features before doing anything else. Could do the same for requestOps

Reply:
It does it here, unless I miss something

Reply:
It does `this.pushCallCounter++;` even though it wouldn't call push :)